api/src/pcapi/routes/backoffice/templates/offerer/get/details/individual_subscription.html

```diff
@@ -112,5 +107,12 @@
       </div>
     </div>
   </form>
+{% else %}
+  <div class="d-flex">
+    <a href="{{ create_dst }}"
+       class="btn btn-md btn-outline-primary fw-bold">
+      <span class="d-none d-xl-inline-block">Commencer le parcours auto-entrepreneur</span>
```

Comment:
Le texte « Envoyer le mail à l’acteur » écrit dans le ticket me semblerait plus clair sur ce qui se produit en cliquant.

Pourquoi cacher le texte du bouton pour les écrans de moins de 1200px de largeur ?

![image](https://github.com/pass-culture/pass-culture-main/assets/23212130/9dfe7937-3b72-43de-9137-d11506ad2ac7)


```suggestion
      Envoyer l'email à l'acteur culturel
```

Reply:
Euh, le span est un mauvais copié-collé, my bad